spacy_experimental/biaffine_parser/tests/test_with_minibatch_by_padded_size.py

```diff
@@ -0,0 +1,37 @@
+from thinc.api import Model, noop
+from spacy_experimental.biaffine_parser.with_minibatch_by_padded_size import (
+    with_minibatch_by_padded_size,
+)
+
+
+def _memoize():
```

Comment:
We can move to a `util` module and reuse it in the `with_splits` tests as well.

Reply:
Done.